ios/RNSScreenComponentView.mm

```diff
@@ -194,10 +241,31 @@ - (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &
   const auto &oldScreenProps = *std::static_pointer_cast<const RNSScreenProps>(_props);
   const auto &newScreenProps = *std::static_pointer_cast<const RNSScreenProps>(props);
 
-  [super updateProps:props oldProps:oldProps];
+  [self setFullScreenSwipeEnabled:newScreenProps.fullScreenSwipeEnabled];
```

Comment:
Why do you sometimes just set the new prop and sometimes you check the old ones before setting new one?